src/Compilers/CSharp/Portable/Utilities/IValueSet.cs

```diff
@@ -51,6 +53,11 @@ internal interface IValueSet
         /// Does this value set contain no values?
         /// </summary>
         bool IsEmpty { get; }
+
+        /// <summary>
+        /// Produce a sample value contained in the set. Throws <see cref="ArgumentException"/> if the set is empty.
```

Comment:
Should document the behavior (return null) when the set is *not* empty but there is no particular value we can return, for example when there is only a value `nint` > `int.MaxValue`.